notebooks/official/migration/sdk-hyperparameter-tuning.ipynb

```diff
@@ -587,10 +503,10 @@
       "source": [
         "#### Task.py contents\n",
         "\n",
-        "In the next cell, you write the contents of the hyperparameter tuning script task.py. I won't go into detail, it's just there for you to browse. In summary:\n",
+        "In the next cell, you write the contents of the hyperparameter tuning script task.py. you won't go into detail, it's just there for you to browse. In summary:\n",
         "\n",
         "- Parse the command line arguments for the hyperparameter settings for the current trial.\n",
-        " - Get the directory where to save the model artifacts from the command line (`--model_dir`), and if not specified, then from the environment variable `AIP_MODEL_DIR`.\n",
+        " - Get the directory where to save the model artifacts from the command line (`--model_dir`), and unspecified, then from the environment variable `AIP_MODEL_DIR`.\n",
```

Comment:
rewrite: "Get the directory where the model artifacts from the command line (`--model_dir`) are to be saved. If unspecified, then from the environment variable `AIP_MODEL_DIR` is used.\n",   [Not sure if this is correct, but the current version isn't clear]